module/scmi_system_power_req/test/mod_scmi_system_power_req_extra.h

```diff
@@ -9,19 +9,6 @@
  */
 #include <mod_scmi_system_power_req.h>
 
-/*! Send an SCMI message */
```

Comment:
these need to stay as they're part of the tests

Reply:
Discard my comment. This is a test related header file, so fine. I'll discard this change.

Reply:
Hmm.. I reconsider my position. 

These 2 prototypes are declared from 2 different header files (here and _mod_scmi_sensor_req_extra.h_). I don't think it is nice. There should be 1 declaration for all sources.
Couldn't _mod_scmi_system_power_req_extra.h_ include _mod_scmi.h_ to get the test setup happy?

Reply:
@etienne-lms : can we move to a separate header mod_scmi_ut.h ? and keep this file under module/scmi/test/includes ? 
scmi_sensor and system power test modules CMakeLists.txt then can include a path for this header. This way it won't affect mod_scmi.h